api/src/modules/analysis/providers/planning-units/adapters/are-puids-allowed-adapter.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import { ArePuidsAllowedPort } from '../are-puids-allowed.port';
+
+import { ScenariosPlanningUnitService } from '../../../../scenarios-planning-unit/scenarios-planning-unit.service';
```

Comment:
this is very much cosmetic and any IDE can deal with this, but can we stick to paths relative to `src`, except within modules where it makes sense to use paths relative to the current file?

Reply:
@hotzevzl to be honest, I would love us to move to `paths` (see tsconfig option, similar to https://docs.nestjs.com/cli/libraries#using-libraries) rather than relative paths. Using `/src...` or `/module...`  breaks `jest`s unit tests.

Reply:
@hotzevzl you may be correct. Will investigate this topic a bit more after/during nestjs-monorepo.